asset/boltdb_manager_test.go

```diff
@@ -0,0 +1,274 @@
+package asset_test
```

Comment:
The tests use logging extensively, but should probably use the methods from the testing package.

See https://golang.org/pkg/testing/#T.Fatalf

It is generally a mistake to use log.Fatal in tests - this will cause the test runner to exit without running any other tests.

Reply:
Thanks. I wasn't sure about half of the things I was doing in testing. I got so used to testify, that I was like "what do I do here again? I will wait and have Eric tell me." <3